hack/test-rune2e.sh

```diff
@@ -92,13 +92,35 @@ function start-scheduler {
         args+=" ${extra_args}"
         log_file="${name}.log"
         for (( i=0; i<${service_num}; i++ )); do
-                sleep 1
+                sleep ${SCHEDULER_START_DELAY}
                 gocmd=" && /usr/local/go/bin/go run resource-management/test/e2e/singleClientTest.go ${args} > ${SIM_LOG_DIR}/${log_file}.$i 2>&1 &"
                 sshcmd="${cmd}${gocmd}"
                 ssh-config "${sshcmd}" "${name}" "${zone}"
         done
 }
 
+# Returns the total number of resourcemanagement managed.
+function get-num-nodes {
+        echo "$((${SIM_NUM}*${SIM_RP_NUM}*${NODES_PER_RP}))"
+}
+
+function get-delay-second {
+  local suggested_delay_second=10
+  if [[ "$(get-num-nodes)" -ge "1000000" ]]; then
+    suggested_delay_second=20
+  fi
+  if [[ "$(get-num-nodes)" -ge "2000000" ]]; then
+    suggested_delay_second=40
+  fi
+  if [[ "$(get-num-nodes)" -ge "5000000" ]]; then
+    suggested_delay_second=180
+  fi
+  if [[ "$(get-num-nodes)" -ge "10000000" ]]; then
+    suggested_delay_second=600
+  fi
+  echo "${suggested_delay_second}"
+}
```

Comment:
Above suggested_delay_second is fixed based on different case, you do not want to provide too much flexibility to let tester input other ${suggested_delay_second}, right?

Reply:
we didn't provide env var for this. tester cannot input this value.